rt/rt.mjs

```diff
@@ -148,6 +148,8 @@ function rt(engine) {
         '%string->symbol': value => 'S' + value,
         '%symbol->string': x => x.substring(1),
 
+        '%string-append': (x, y) => 's' + x.substring(1) + y.substring(1),
```

Comment:
I would think we'd need the same `String()` hack here

Reply:
Good catch. Done.